src/OrchardCore.Themes/TheAdmin/Views/Message.cshtml

```diff
@@ -1,4 +1,8 @@
 @{
+    if (Model.Type == null)
+    {
+        return;
```

Comment:
Can we directly return from a piece of razor code? Never did this ;)

Reply:
I guess we learn something new everyday :)   I use return in view which help clean up logic a bit